index.html

```diff
@@ -183,11 +183,7 @@ <h1 id="logo-wrap">
 <div class="row">
 <div class="col-lg-12">
 <div class="col-sm-6 col-xs-12" id="banner-title"><span>Apache MXNet</span>
-<p id="landing-title">is a flexible and efficient library for deep learning.</p>
-<p id="intro">
-                Building a high-performance deep learning library requires many system-level design decisions. In this design note, we share the rationale for the specific 
-                choices made when designing MXNet. We imagine that these insights may be useful to both deep learning practitioners and builders of other deep learning systems.
-            </p>
+<p id="landing-title">flexible and efficient library for deep learning.</p>
```

Comment:
I think it will be good to keep this as below

Apache MXNet
A flexible and efficient library for deep learning.

Thoughts?